flowzone.yml

```diff
@@ -75,10 +86,10 @@
     with:
       registry: ghcr.io
       username: ${{ env.GHCR_USER || github.actor }}
-      password: ${{ secrets.GHCR_TOKEN || secrets.FLOWZONE_TOKEN }}
+      password: ${{ steps.gh_token.outputs.token || secrets.FLOWZONE_TOKEN }}
```

Comment:
If we are deprecating this secret we need to be clear about that. I thought it was preferable to allow the option for a different token with package write access. Maybe that's not required, but we should be clear that the secret is unused going forward.

Reply:
I don't think there is a goods reason to keep it, I'll clean up the docs.